src/components/aside/WorkspaceInfo.vue

```diff
@@ -13,9 +13,26 @@
       </div>
       <router-link
         class="workspace-info__settings-link"
-        :to="{ name: 'workspace-settings', params: {workspaceId: workspace.id} }"
+        :to="{
+          name: 'workspace-settings',
+          params: { workspaceId: workspace.id },
+        }"
       >
-        {{ $t('workspaces.settings.label') }}
+        {{ $t("workspaces.settings.label") }}
+      </router-link>
+    </div>
+    <div
+      class="workspace-info__event-indicator"
+      @mouseover="eventsIndicatorMouseover"
+      @mouseleave="eventsIndicatorMouseleave"
+    >
+      <router-link
+        :to="{
+          name: 'workspace-settings-billing',
+          params: { workspaceId: workspace.id },
+        }"
+      >
+        <CircleProgress :current="eventsCount" :max="plan.eventsLimit || 0" />
       </router-link>
```

Comment:
Also add `v-if= "isAdmin"`. Because you can't go to the billing page without admin rights